docs/reference/ml/apis/dfanalyticsresources.asciidoc

```diff
@@ -0,0 +1,114 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-dfanalytics-resources]]
+=== {dfanalytics-cap} resources
+
+{dfanalytics-cap} resources relate to APIs such as <<put-dfanalytics>> and
+<<get-dfanalytics>>.
```

Comment:
Since those APIs are related to "data frame analytics jobs" should this also be "Data frame analytics job resources"?